scram-project-build.file

```diff
@@ -285,7 +285,7 @@ for DIR in $ELF_DIRS $DROP_SYMBOLS_DIRS; do
     if [ $(echo $ELF_BINS | wc -w) -gt 1 ] ; then
       dwz -m .debug/common-symbols.debug -M common-symbols.debug $ELF_BINS
     fi
-    echo "$ELF_BINS" | xargs -t -n1 -P%{compiling_processes} -I% sh -c 'objcopy --compress-debug-sections --only-keep-debug % .debug/%.debug; objcopy --strip-debug --add-gnu-debuglink=.debug/%.debug %'
+    echo "$ELF_BINS" | xargs -t -n1 -I% sh -c 'objcopy --compress-debug-sections --only-keep-debug % .debug/%.debug; objcopy --strip-debug --add-gnu-debuglink=.debug/%.debug %'
```

Comment:
@apfeiffer1 , on which system did you test it? this section should only run for slc*|fc* archs and I think -P option works there